src/test/java/Homework19.java

```diff
@@ -0,0 +1,32 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class Homework19 extends BaseTest{
+
+    @Test(enabled = true)
+    public void testDeletePlaylist()throws InterruptedException{
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Reply:
I deleted sleep everywhere, but skip InterruptedException. I fixed it.